Frontend/src/setupTests.js

```diff
@@ -0,0 +1,3 @@
+import { configure } from "enzyme";
```

Comment:
Can we have this in the frontend tests folder itself instead of one level above?